Library/Formula/arangodb.rb

```diff
@@ -14,9 +14,17 @@ class Arangodb < Formula
 
   depends_on "go" => :build
   depends_on "openssl"
+  depends_on "readline"
 
   needs :cxx11
 
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!